semcore/bulk-textarea/src/translations/en.json

```
@@ -0,0 +1,10 @@
+{
+  "BulkTextarea.ClearAllButton.buttonText": "Clear all",
+  "BulkTextarea.Counter.ofAllowedRows:sr-message": "of {rowsNumber} rows",
```

```suggestion
  "BulkTextarea.Counter.ofAllowedRows:sr-message": "of {rowsNumber} lines",
```
we should probably replace rows with lines here too (and update translations of this string)